### PR TITLE
Logger and web UI used in first hatch

### PR DESCRIPTION
**Pull request by abencz**
particle_logger.py polls the particle API for temperature/RH and logs
data to .csv files on disk. quail_maker_web.py is a Flask app that reads
the logged CSV data and presents it as a simple web page. It also allows
all data to be downloaded in a combines .csv file.

### CODE REVIEW

File: scripts/particle_logger.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-f", "--file-prefix", type=str, default="quail_maker")
+    parser.add_argument("-d", "--log-delay", type=int, default=300)
+
+    return parser.parse_args()
+
+
+def main():
+    device_id = "<tbd>"
+    access_token = "<tbd>"
```
Comment (abencz) [inline scripts/particle_logger.py]: I took these out before adding to version control. They need to be loaded either from file or via argparse.

File: scripts/quail_maker_web.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                data.append(row)
+
+    return data
+
+
+def row_to_html(row):
+    return f"""
+    <tr>
+        <td>{row["timestamp"].isoformat()}</td>
+        <td><center>{row["temperature"]:.1f} \xb0C</center></td>
```
Comment (abencz) [inline scripts/quail_maker_web.py:39]: `\xb0` is the degree symbol: "º"

File: scripts/quail_maker_web.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    files = glob(path)
+    return sorted(files)
+
+
+def gather_data(directory="./"):
+    files = get_csv_files(directory)
+
+    data = []
+
+    edt = timezone(timedelta(hours=-4))
```
Comment (abencz) [inline scripts/quail_maker_web.py:22]: Hardcoded for EDT for now. Would be better to get this from a library.

Might need to change to EST for second hatch?

File: scripts/quail_maker_web.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+import csv
+from datetime import datetime, timedelta, timezone
+from flask import Flask, send_file
+from glob import glob
+from tempfile import NamedTemporaryFile
+
+
+def get_csv_files(directory):
+    path = f"{directory}/*.csv"
```
Comment (abencz) [inline scripts/quail_maker_web.py]: Really should use `os.path` to join the path here.

**Review by vanessanielsen — APPROVED** [review]

File: scripts/quail_maker_web.py (hunk 3 above)
Comment (vanessanielsen) [reply scripts/quail_maker_web.py:22]: Yes

**Comment by abencz** [comment]
Getting the timezone using Python is going to take some more investigation.

**Pull request merged**